common.go

```diff
@@ -600,6 +601,9 @@ type Config struct {
 	// Allow short headers (experimental, and only applies to server).
 	AllowShortHeaders bool
 
+	// If true disables use of the Extended Master Secret extension
```

Comment:
Higher level point: why would a user of `crypto/tls` want to (and know to) disable EMS?